jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -377,9 +377,10 @@ properties:
             type: string
             enum: [NRCA1, NRCA2, NRCA3, NRCA4, NRCALONG, NRCB1, NRCB2, NRCB3, NRCB4,
               NRCBLONG, NRS1, NRS2, ANY, MIRIMAGE, MIRIFULONG, MIRIFUSHORT,
-              NIS, GUIDER1, GUIDER2, N/A]
+              NIS, GUIDER1, GUIDER2, N/A, MULTIPLE]
             fits_keyword: DETECTOR
             blend_table: True
+            blend_rule: multi
```

Comment:
While we're at it, we should make the same addition to the `module` keyword, because data from multiple modules can be combined into a single image.